src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+	import type { WizardStep } from '@dfinity/gix-components';
+	import { getContext } from 'svelte';
+	import BtcSendForm from '$btc/components/send/BtcSendForm.svelte';
+	import BtcSendProgress from '$btc/components/send/BtcSendProgress.svelte';
+	import BtcSendReview from '$btc/components/send/BtcSendReview.svelte';
+	import { SEND_CONTEXT_KEY, type SendContext } from '$icp-eth/stores/send.store';
+	import SendQrCodeScan from '$lib/components/send/SendQRCodeScan.svelte';
+	import {
+		btcAddressMainnet,
+		btcAddressRegtest,
+		btcAddressTestnet
+	} from '$lib/derived/address.derived';
+	import { WizardStepsSend } from '$lib/enums/wizard-steps';
+	import type { NetworkId } from '$lib/types/network';
+	import { isNetworkIdBTCRegtest, isNetworkIdBTCTestnet } from '$lib/utils/network.utils';
+	import { decodeQrCode } from '$lib/utils/qr-code.utils';
+
+	export let currentStep: WizardStep | undefined;
```

Comment:
not sure why the `currentStep` is exposed?

Reply:
`currentStep` will be controlled by the parent component called `BtcSendModal` (the next PR). This is how it's done in `IcSendModal` - https://github.com/dfinity/oisy-wallet/blob/941024a552be4cb4378c9ca3862df7d986cf59e6/src/frontend/src/icp/components/send/IcSendModal.svelte#L55

Reply:
Ah gotcha! I think I reviewed it thinking we were one level up here. You can probably ignore all my comments.